test/network_test.go

```diff
@@ -67,6 +67,9 @@ func (s *CLITestSuite) TestNetworkList() {
 	tests := []CLITest{
 		{args: "network list", fixture: "network/list.golden"},
 		{args: "network list --output json", fixture: "network/list-json.golden"},
+		{args: "network list --name prod-gcp-us-central1,prod-aws-us-east1 --cloud aws", fixture: "network/list-name-cloud.golden"},
+		{args: "network list --region eastus2 --cidr 10.0.0.0/16", fixture: "network/list-region-cidr.golden"},
+		{args: "network list --phase READY --connection-types transitgateway,peering", fixture: "network/list-phase-connection.golden"},
```

Comment:
are `--cloud` and `--phase` case sensitive?

just wanted to double check since you used upper case for "AWS` below.

Reply:
We like to accept both, but we only use lowercase in our examples as a way to standardize our formatting. It would be great to accept both but change all of these to lowercase!
```suggestion
		{args: "network list --name prod-gcp-us-central1,prod-aws-us-east1 --cloud aws", fixture: "network/list-name-cloud.golden"},
		{args: "network list --region eastus2 --cidr 10.0.0.0/16", fixture: "network/list-region-cidr.golden"},
		{args: "network list --phase ready --connection-types transitgateway,peering", fixture: "network/list-phase-connection.golden"},
```

Reply:
updated flags to lowercase